tests/containers/bci_tests.pm

```diff
@@ -0,0 +1,158 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: bci-tests runner
+#   SUSE Linux Enterprise Base Container Images (SLE BCI)
+#   provides truly open, flexible and secure container images and application
+#   development tools for immediate use by developers and integrators without
+#   the lock-in imposed by alternative offerings.
+#   - install needed python environment
+#   - clone bci-tests repo
+#   - build project
+#   - run test for each given environment
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base qw(consoletest);
+use XML::LibXML;
+use utils qw(systemctl zypper_call ensure_ca_certificates_suse_installed);
+use version_utils qw(get_os_release);
+use containers::common;
+use testapi;
+
+
+sub parse_logs {
+    my ($self, $args) = @_;
+
+    upload_logs('junit_build_serial.xml');
+    upload_logs('junit_build_parallel.xml');
+
+    # bci-tests produce separate XUnit results files for each environment.
+    # We need tp merge all together into a single xml file that will
+    # be used by OpenQA to represent the results in "External results"
+    my $dom  = XML::LibXML::Document->new('1.0', 'utf-8');
+    my $root = $dom->createElement('testsuites');
+    $dom->setDocumentElement($root);
+
+    # Dump xml contents to a location where we can access later using data_url
+    for my $env (split(/,/, $args->{envs})) {
+        upload_logs('junit_' . $env . '_serial.xml');
+        my $log_file = upload_logs('junit_' . $env . '_parallel.xml');
+        my $dom      = XML::LibXML->load_xml(location => "ulogs/$log_file");
+        for my $node ($dom->findnodes('//testsuite')) {
+            # Replace default attribute name "pytest" by its env name
+            $node->{name} =~ s/pytest/$env/;
+            # Append test results to the resulting xml file
+            $root->appendChild($node);
+        }
+    }
+    $dom->toFile("os-autoinst-distri-opensuse/data/result.xml", 1);
+    # Download file from host pool to the instance
+    assert_script_run 'curl -f -v ' . data_url('result.xml') . " > /tmp/result.txt";
+    parse_extra_log('XUnit', '/tmp/result.txt');
```

Comment:
```suggestion
    $dom->toFile('result.xml', 1);
    # Download file from host pool to the instance
    assert_script_run('(cd /tmp/ && curl -O ' . autoinst_url('/files/result.xml') .  ')');
    parse_extra_log('XUnit', '/tmp/result.txt');
```

I think this is more common.

Reply:
I changed it, thanks for the suggestion :)